### PR TITLE
Internalize vue-avatar to ODS, improve accessibility & support lazy img loading

### PR DESCRIPTION
**Pull request by pascalwengerter**

### CODE REVIEW

File: src/components/avatars/OcAvatar.vue [hunk 1]
```diff
@@ -48,18 +64,126 @@ export default {
...
+     */
     width: {
       type: Number,
       required: false,
       default: 50,
     },
+    /**
+     * The initials get computed by the component itself
+     */
+    initials: {
```
Comment (kulmann) [inline src/components/avatars/OcAvatar.vue:78]: Why do we need the initials as a prop? From my understanding we can have the initials of the userName as a computed property. IMO the parser is not needed for us.

**Review by kulmann — CHANGES_REQUESTED** [review]

File: src/components/avatars/OcAvatar.vue (hunk 1 above)
Comment (pascalwengerter) [reply src/components/avatars/OcAvatar.vue]: Successfully re-re-rebased this and implemented the feedback, ready for re-review

File: src/components/OcImage.vue [hunk 2]
```diff
@@ -36,6 +36,14 @@ export default {
       required: false,
       default: null,
     },
+    loadingType: {
```
Comment (kulmann) [inline src/components/OcImage.vue:39]: Loading type is not applied to the `img` tag in the template

File: src/components/avatars/OcAvatar.vue [hunk 3]
```diff
@@ -1,31 +1,50 @@
...
+    class="vue-avatar--wrapper oc-avatar"
+    :style="[style]"
+    :width="width"
     :aria-label="accessibleLabel === '' ? null : accessibleLabel"
     :aria-hidden="accessibleLabel === '' ? 'true' : null"
     :focusable="accessibleLabel === '' ? 'false' : null"
     :role="accessibleLabel === '' ? null : 'img'"
-  />
+  >
+    <oc-img v-if="isImage" loading="lazy" class="avatarImg" :src="src" @error="onImgError" />
```
Comment (kulmann) [inline src/components/avatars/OcAvatar.vue:11]: prop name is not `loading`, but `loading-type`

File: src/components/avatars/OcAvatar.vue [hunk 4]
```diff
@@ -1,31 +1,50 @@
 <template>
-  <avatar
-    class="oc-avatar"
-    :username="userName"
-    :size="width"
-    :src="src"
+  <span
+    class="vue-avatar--wrapper oc-avatar"
+    :style="[style]"
```
Comment (kulmann) [inline src/components/avatars/OcAvatar.vue:4]: Why are the styles wrapped into an array? 🤔

**Review by kulmann — CHANGES_REQUESTED** [review]

File: src/components/avatars/OcAvatar.vue (hunk 4 above)
Comment (pascalwengerter) [reply src/components/avatars/OcAvatar.vue:4]: Leftovers from the vue-avatar copy-pasta, thanks I'll get rid of it

File: src/components/OcImage.vue (hunk 2 above)
Comment (pascalwengerter) [reply src/components/OcImage.vue:39]: Might've lost this one during the rebases, brining it back now 💪🏽

Comment (kulmann) [reply src/components/OcImage.vue:42]: ❤️

**Review by kulmann — APPROVED** [review]
Awesome! 🥇

**Pull request merged**